common/src/main/java/tachyon/ClientBase.java

```diff
@@ -287,7 +289,8 @@ protected synchronized InetSocketAddress getAddress() {
    *         been called before calling this method or during the retry
    * @throws ConnectionFailedException if network connection failed
    */
-  protected <V> V retryRPC(RpcCallable<V> rpc) throws IOException, ConnectionFailedException {
+  protected synchronized <V> V retryRPC(RpcCallable<V> rpc) throws IOException,
```

Comment:
Why these two `retryRPC` methods need to be `synchronized` since `connect` has been synchronized?